tools/topology/topology2/include/common/tokens.conf

```diff
@@ -103,6 +103,8 @@ Object.Base.VendorToken {
 	"sof_tkn_stream" {
 		playback_compatible_d0i3         "1200"
 		capture_compatible_d0i3          "1201"
+		playback_delay                   "1202"
+		capture_delay                    "1203"
```

Comment:
What does this represent? The delay is a dynamic value that should be calculated, I have no idea why this would be added in topology.

The delay that we can have in the topology is really the size of the host DMA buffer, but I don't see how this is related to your suggestion. Put differently, the buffer is defined in terms of bytes or ms at the host copier level. and the delay will be the sum of all the internal delays for the pipeline.

Reply:
I didn't get your comments and discussed with Bard, then we got an idea that we can add a token.  yes, the delay is dynamic and I also feel uncertain to add it in topology. Thanks for you explanation.